### PR TITLE
factor out transposition_transposase_source

### PR DESCRIPTION
**Pull request by mccalluc**
Use the suggest-edit mechanism to indicate which is the correct definition for this field; Chuck will followup and regenerate docs.

### CODE REVIEW

File: src/ingest_validation_tools/table-schemas/assays/bulkatacseq-v1.yaml [hunk 1]
```diff
@@ -66,8 +66,5 @@ fields:
     required: False
 - name: transposition_method
   description: Modality of capturing accessible chromatin molecules. The kit used, for example.
-- name: transposition_transposase_source
-  description: The source of the Tn5 transposase and transposon used for capturing
-    accessible chromatin.
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/assays/bulkatacseq-v1.yaml:71]: description: The source of the Tn5 transposase used for capturing accessible chromatin.

File: src/ingest_validation_tools/table-schemas/assays/bulkatacseq-v0.yaml [hunk 2]
```diff
@@ -66,8 +66,5 @@ fields:
     required: False
 - name: transposition_method
   description: Modality of capturing accessible chromatin molecules. The kit used, for example.
-- name: transposition_transposase_source
-  description: The source of the Tn5 transposase and transposon used for capturing
-    accessible chromatin.
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/assays/bulkatacseq-v0.yaml:71]: description: The source of the Tn5 transposase used for capturing accessible chromatin.

File: src/ingest_validation_tools/table-schemas/assays/scatacseq-v0.yaml [hunk 3]
```diff
@@ -58,13 +58,7 @@ fields:
       - bulkATACseq
       - snATACseq
       - sciATACseq
-- name: transposition_transposase_source
-  description: The source of the Tn5 transposase and transposon used for capturing accessible chromatin.
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/assays/scatacseq-v0.yaml:62]: description: The source of the Tn5 transposase used for capturing accessible chromatin.

File: src/ingest_validation_tools/table-schemas/assays/scatacseq-v1.yaml [hunk 4]
```diff
@@ -58,13 +58,7 @@ fields:
       - bulkATACseq
       - snATACseq
       - sciATACseq
-- name: transposition_transposase_source
-  description: The source of the Tn5 transposase and transposon used for capturing accessible chromatin.
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/assays/scatacseq-v1.yaml:62]: description: The source of the Tn5 transposase used for capturing accessible chromatin.

File: src/ingest_validation_tools/table-schemas/includes/fields/transposition_transposase_source.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# bulkatacseq-v0.yaml; bulkatacseq-v1.yaml
+- name: transposition_transposase_source
+  description: The source of the Tn5 transposase and transposon used for capturing
+    accessible chromatin.
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/includes/fields/transposition_transposase_source.yaml:4]: description: The source of the Tn5 transposase used for capturing accessible chromatin.

File: src/ingest_validation_tools/table-schemas/includes/fields/transposition_transposase_source.yaml [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+
+# scatacseq-v0.yaml; scatacseq-v1.yaml
+- name: transposition_transposase_source
+  description: The source of the Tn5 transposase and transposon used for capturing accessible chromatin.
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/includes/fields/transposition_transposase_source.yaml:9]: description: The source of the Tn5 transposase used for capturing accessible chromatin.

**Comment by mccalluc** [comment]
@cebriggs7135 -- To make this tractable, we need to use the suggest edits mechanism in github. Let's do a screen share after the curation meeting.

File: src/ingest_validation_tools/table-schemas/includes/fields/transposition_transposase_source.yaml [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+  constraints:
+    enum:
+      - 10X snATAC
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/includes/fields/transposition_transposase_source.yaml:12]: This enum list is correct for sc/sn ATACseq only. Chris will follow-up on bulk ATACseq input for this field (review already submitted metadata for bulkATAcseq to check).

**Pull request closed without merging**